applications/sandbox/systems/physx_test_system.hpp

```diff
@@ -52,7 +130,7 @@ namespace legion::physics
 
         void suzzaneRainTick(legion::time::span deltaTime);
 
-        void createCubeStack(const math::vec3& extents, size_t stackSize, const math::vec3& startPos);
+        void createCubeStack(const math::vec3& extents, size_t stackSize, const math::vec3& startPos,int stopAfterStack = -1);
```

Comment:
int here would be better as size_type, but this is test system anyways so i don't rly care much